rubocop-rspec.yml

```diff
@@ -0,0 +1,29 @@
+RSpec:
+  Enabled: false
```

Comment:
I think we should go with the opposite, e.g. enable and remove the cops below (which would be enabled -- they're so by default), if only to be consistent with the approach we're taking with the `rubocop-rails.yml` file.
Do we know of rspec cops that teams have been _disabling_ explicitly?

Reply:
updated b56326ed379a85082d16e1cd0a9d4819de2428d6